develop/interoperability/send-messages.md

```diff
@@ -0,0 +1,44 @@
+---
+title: XCM - Send Messages
```

Comment:
Maybe for SEO purposes, this could be better; idk
@eshaben or @dawnkelly09 can definitely help with this comment for sure

```suggestion
title: Send XCM Messages
```

Reply:
Agree with Nico here